website/docs/contributing/general/ADR/domain-language.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "ADR: Domain language"
+---
+
+## Background
+
+In the codebase, we have seen the need to define a domain language that we use to refer to features, methods to keep it consistent across the codebase. This ADR will contain a growing list of domain language used to keep the consistency across the codebase.
+
+## Decision
+
+We have decided to use the same domain language for the features we develop. Each feature will have it's own domain language to keep it consistent across the codebase.
+
+## Change requests domain language
+
+* *Change request*
+
+An entity referring to the overarching data structure of a change request. A change request contains changes, and can be approved or rejected.
```

Comment:
We can use lists for listing terms, but please make sure they're treated as a single list in that case, and not as multiple lists (as is the case right now). Optionally, use a [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) (sadly, markdown doesn't have anything that maps directly to it, so you'll need to mark it up with raw html in that case).

If you just want to turn it into a single list, consider placing the term and the definition on the same line, separated by a colon.

```suggestion
* *Change request*: An entity referring to the overarching data structure of a change request. A change request contains changes, and can be approved or rejected.
```